core/utils.js

```diff
@@ -587,6 +587,68 @@ Blockly.utils.getBlockTypeCounts = function(block, opt_stripFollowing) {
   return typeCountsMap;
 };
 
+var resizeFromKeyboard = true;
+var initialWidth = 0; // The width of the window before resizing started
+var lastHeight = 0; // The height at each frame of the resize
+var debounceTimeout = null;
+
+/**
+ * Called in the resize event handler to sense if the source
+ * is from an on screen keyboard or user.
+ * @param {!Blockly.WorkspaceSvg} workspace Any workspace in the SVG.
+ */
+Blockly.utils.resizeTracker = function(workspace) {
+  if (debounceTimeout) {
+    clearTimeout(debounceTimeout);
+  }
+  debounceTimeout = setTimeout(function(){
+    var injectionDiv = workspace.getInjectionDiv();
+    resizeFromKeyboard = true;
+    initialWidth = injectionDiv.clientWidth;
```

Comment:
does this need to also be in an `if (injectionDiv)` check?

Reply:
fixed!